distribution/lib/Standard/Base/0.0.0-dev/src/System/File_Format_Metadata.enso

```diff
@@ -50,7 +50,7 @@ type File_Format_Metadata
     guess_extension self = self.extension.if_nothing <|
         self.name.if_not_nothing <|
             guessed = find_extension_from_name self.name
-            if guessed == "" then Nothing else guessed
+            if guessed == "" then Nothing else guessed.to_case ..Lower
```

Comment:
Note that this may not be enough. If the extension is specified directly in `File_Format_Metadata` (the `self.extension != Nothing` case) then it is not lowercased.

You'd need to update places where the extension is explicitly created from the file, e.g. the `from` conversion just a few lines below: `File_Format_Metadata.from (that:File)`, or change `guess_extension` to also apply the lowercasing to the `self.extension` path.

Otherwise it seems currently that in some cases the extensions are lowercased but not in others.